lib/ex_brasil_api/application.ex

```diff
@@ -7,9 +7,23 @@ defmodule ExBrasilApi.Application do
 
   @impl true
   def start(_type, _args) do
+    pools =
```

Comment:
Uma coisa que ainda preciso estudar, aqui nesse projeto estamos iniciando o Finch na árvore de supervisão, mas e quando outro projeto for usar esse nosso, vamos precisar adicionar a nossa lib na árvore do outro projeto? Com isso tudo vai ser iniciado automáticamente?